From 2a8c54f03ee0a36adeed7d2dc7534cecd6dfa3b9 Mon Sep 17 00:00:00 2001
From: Li Jun <jun.li@freescale.com>
Date: Thu, 26 Nov 2015 10:48:05 +0800
Subject: [PATCH] MLK-11896-1 usb: chipidea: imx: add break for switch case

Fix the coverity check complains although there is no function issue
with current code.

Signed-off-by: Li Jun <jun.li@freescale.com>
(cherry picked from commit 56938c1864e739b1e833aee210b18e8df23d3846)
---
 drivers/usb/chipidea/ci_hdrc_imx.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/usb/chipidea/ci_hdrc_imx.c b/drivers/usb/chipidea/ci_hdrc_imx.c
index dc7c9f1..66dfcfc 100644
--- a/drivers/usb/chipidea/ci_hdrc_imx.c
+++ b/drivers/usb/chipidea/ci_hdrc_imx.c
@@ -307,10 +307,12 @@ static int ci_hdrc_imx_notify_event(struct ci_hdrc *ci, unsigned event)
 		if (data->usbmisc_data)
 			return imx_usbmisc_term_select_override(
 					data->usbmisc_data, true, 1);
+		break;
 	case CI_HDRC_IMX_TERM_SELECT_OVERRIDE_OFF:
 		if (data->usbmisc_data)
 			return imx_usbmisc_term_select_override(
 					data->usbmisc_data, false, 0);
+		break;
 	default:
 		dev_dbg(dev, "unknown event\n");
 	}
-- 
1.8.0

